src/woo_search/index/api/serializers/publications.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+
+class DocumentSerializer(serializers.Serializer):
+    uuid = serializers.UUIDField()
+    publicatie = serializers.UUIDField()
+    publisher = serializers.UUIDField()
+    identifier = serializers.CharField(max_length=255, required=False)
+    officiele_titel = serializers.CharField(max_length=255)
+    verkorte_titel = serializers.CharField(max_length=255, required=False)
```

Comment:
```suggestion
    verkorte_titel = serializers.CharField(max_length=255, required=False, blank=True)
```
it's probably much more likely that we'll send empty values rather than omit the keys.